assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -354,7 +354,11 @@ che.infra.kubernetes.ingress.domain=
 che.infra.kubernetes.namespace=
 
 # Defines Kubernetes Service Account name which should be specified to be bound to all workspaces pods.
-# Note that Che Server won't create the service account and it should exist.
+# Note that Kubernetes Infrastructure won't create the service account and it should exist.
+# OpenShift infrastructure will check if project is predefined(if `che.infra.openshift.project` is not empty):
+#  - if it is predefined then service account must exist there
+#  - if it is 'NULL' or empty string then infrastructure will create new OpenShift project per workspace
```

Comment:
This is for single-user only, right? I'm not clear on how this would work in the multi-user case.

Reply:
This is for both. But out-of-the-box it will work only with OAuth provider configured, without it Che SA needs extra permissions for single or multi user mode.